server/service/src/sync/mod.rs

```diff
@@ -79,3 +80,31 @@ impl ActiveStoresOnSite {
         self.stores.iter().map(|r| r.store_row.id.clone()).collect()
     }
 }
+
+#[derive(Error, Debug)]
+pub(crate) enum GetInactiveStoresOnSiteError {
+    #[error("Database error while getting inactive store on site")]
+    DatabaseError(RepositoryError),
+    #[error("Site id is not set in database")]
+    SiteIdNotSet,
+}
+
+pub(crate) fn get_inactive_store_ids(
```

Comment:
I think this logic should be in another PR/branch, to be done when we implement store switch (it needs extensive check and having it here now would mean it can potentially have a use case we haven't thought about/tested)

Reply:
Sorry, I had already made a [second PR](https://github.com/openmsupply/open-msupply/pull/943) with just the other changes! I'll rebase this one to stack it correctly.

Reply:
Done. Weird, the diff isn't picking up that the new base PR has most of the commits in this PR...